docs/zh/sql-reference/sql-statements/table_bucket_part_index/ALTER_TABLE.md

```diff
@@ -565,18 +565,47 @@ ALTER TABLE orders ORDER BY (dt,revenue,state);
 
 ```sql
 -- 增加字段
-ALTER TABLE [<db_name>.]<tbl_name>
-MODIFY COLUMN <column_name> ADD FIELD <field_name>  <field_type> [AFTER <prior_field_name> |FIRST]
+ALTER TABLE [<db_name>.]<tbl_name> MODIFY COLUMN <column_name>
+ADD FIELD field_path field_desc
 
 -- 删除字段
-ALTER TABLE [<db_name>.]<tbl_name>
-MODIFY COLUMN <column_name> DROP FIELD <field_name>
+ALTER TABLE [<db_name>.]<tbl_name> MODIFY COLUMN <column_name>
+DROP FIELD field_path
+
+field_path ::= [ { <top_field_name>. | [*]. } [ ... ] ]<field_name>
+
+field_desc ::= <field_type> [ AFTER <prior_field_name> | FIRST ]
 ```
 
 参数：
 
-- `field_name`：需要增加或删除字段的名称。可以是单独的字段名，表示第一层级的字段，例如 `new_field_name`。也可以是 Column Access Path，用以表示嵌套层级的字段，例如 `lv1_k1.lv2_k2.new_field_name`。
-- `prior_field_name`：新增字段的前一个字段。与 AFTER 关键字合用，可以表示新加字段的顺序。如果指定 FIRST 关键字，表示新增第一个字段，则无需指定该参数。`prior_field_name` 的层级由 `field_name` 决定，无需手动指定。
+- `field_path`：需要增加或删除字段。可以是单独的字段名，表示第一层级的字段，例如 `new_field_name`。也可以是 Column Access Path，用以表示嵌套层级的字段，例如 `lv1_k1.lv2_k2.[*].new_field_name`。
+  - `top_field_name`：Column Access Path 中的第一层级字段。
+  - `[*]`：当 ARRAY 和 STRUCT 类型发生嵌套时，`[*]` 代表操作 ARRAY 字段中的所有元素，用于在 ARRAY 类型字段下嵌套的每个 STRUCT 类型字段中添加或删除子字段。
+  - `field_name`：需要增加或删除字段的名称。
```

Comment:
`field_name` means the last field of field_path must be a simple field name, it can't be `[*]`, such as `lv1_k1.lv2_k2.[*]`, it's wrong.

Reply:
And, the syntax is clear, in my opinion.